content/zh/docs/tutorials/hello-minikube.md

```diff
@@ -22,64 +21,48 @@ menu:
     title: "Get Started"
     weight: 10
     post: >
-      <p>Ready to get your hands dirty? Build a simple Kubernetes cluster that runs "Hello World" for Node.js.</p>
+      <p>Ready to get your hands dirty? Build a simple Kubernetes cluster that runs a sample app.</p>
 card:
   name: tutorials
   weight: 10
----
 -->
 
 <!-- overview -->
 
 <!--
-This tutorial shows you how to run a simple Hello World Node.js app
+This tutorial shows you how to run a sample app
 on Kubernetes using [Minikube](/docs/setup/learning-environment/minikube) and Katacoda.
 Katacoda provides a free, in-browser Kubernetes environment.
 -->
-本教程向您展示如何使用 [Minikube](/zh/docs/setup/learning-environment/minikube) 和 Katacoda 在 Kubernetes 上运行一个简单的 “Hello World” Node.js 应用程序。Katacoda 提供免费的浏览器内 Kubernetes 环境。
+本教程向你展示如何使用 [Minikube](/zh/docs/setup/learning-environment/minikube) 和 Katacoda
+在 Kubernetes 上运行一个应用示例。Katacoda 提供免费的浏览器内 Kubernetes 环境。
 
-{{< note >}}
 <!--
 You can also follow this tutorial if you've installed [Minikube locally](/docs/tasks/tools/install-minikube/).
 -->
-如果您已在本地安装 [Minikube](/zh/docs/tasks/tools/install-minikube/)，也可以按照本教程操作。
-
+{{< note >}}
+如果你已在本地安装 [Minikube](/zh/docs/tasks/tools/install-minikube/)，也可以按照本教程操作。
 {{< /note >}}
 
 
-
 ## {{% heading "objectives" %}}
 
-
 <!--
 * Deploy a hello world application to Minikube.
```

Comment:
Deploy a sample application to Minikube.

Reply:
```
Deploy a sample application to Minikube.
Run the app.
View application logs.
```

Reply:
yes. Fixed.